cmd/fossa/report.go

```diff
@@ -60,7 +60,9 @@ func reportLicenses(s *spinner.Spinner, endpoint, apiKey string, analyses []anal
 		log.Logger.Fatalf("Invalid API endpoint: %s", err.Error())
 	}
 
-	s.Suffix = " Loading licenses..."
+	if format == "text" {
```

Comment:
string `text` has 4 occurrences, make it a constant

Reply:
TY. Should be fixed now